tests/s3/copy_object/test_parallel_copy_object.py

```diff
@@ -279,17 +315,19 @@ def test_44792(self):
         LOGGER.info("STARTED: Test Parallel Self copy and put object in same bucket "
                     "(simple and multipart source object)")
         LOGGER.info("Upload object1 to source bucket1")
-        self.create_put_object("src-bucket", "obj1", 5 * 1024)
+        self.create_put_object(self.src_bkt, self.key_obj1, 5 * 1024)
         self.parallel_copy_and_put_object(
-            ("src-bucket", "obj1", "src-bucket", "obj1", "InvalidRequest"),
-            ("src-bucket", "obj2", self.file_path))
+            (self.src_bkt, self.key_obj1, self.src_bkt, self.key_obj1, "InvalidRequest"),
+            (self.src_bkt, self.key_obj2, self.file_path))
         LOGGER.info("Upload multipart object obj3 to src-bucket")
-        self.s3mp_obj.simple_multipart_upload("src-bucket", "mp-obj1", 3 * 1024, self.file_path, 4)
+        self.s3mp_obj.simple_multipart_upload(self.src_bkt, self.key_mpobj1, 3 * 1024,
+                                              self.file_path, 4)
         self.parallel_copy_and_put_object(
-            ("src-bucket", "mp-obj1", "src-bucket", "mp-obj1", "InvalidRequest"),
-            ("src-bucket", "obj4", self.file_path))
+            (self.src_bkt, self.key_mpobj1, self.src_bkt, self.key_mpobj1, "InvalidRequest"),
```

Comment:
Let's use the constants for error messages in the error_messages module and not incorporate string literals in the test

Reply:
Its coming from @kapiljinna 's PR: https://github.com/Seagate/cortx-test/pull/1813/files. So we can do one that is merged. Or I Can comment in his PR to accommodate this change once this PR is merged.